ventilator_alarm.py

```diff
@@ -46,6 +46,11 @@ def run(self, name):
                     self.time_last_kick_received == cur_time
                     if msg['val'] != 0:
                         self.request_queue.put({'type': 'error', 'value': msg['val']})
+                elif msg['type'] == "PRES":
+                    if ((msg['val'] > proto.settings_values['PK'] + proto.settings_values['ADPK']) or
+                       (msg['val'] < proto.settings_values['PK'] - proto.settings_values['ADPK'])):
+                        self.request_queue.put({'type': 'alarm', 'priority': 42, 'value': 5})
```

Comment:
how is the priority used? Does this make sense given that priority queues are not available for multiprocessing?

Reply:
In my opinion the priority can have a sense. But it depends on what is the final design around alarm. We can imagine a design where a alarm code has its own sound or visual and if several alarms are triggered at the same time, the higher priority can be played first, and when it is fixed the second one, etc.
At least it is easy to remove.

For the value, yes it is just for test, ideally we should assign a name to a value and having a list of alarms somewhere (in protocol.py ?). I made the pull request to get a feedback and not really to merge it yet, just to define a bit more what is expected and if the concept is right.